FitNesseRoot/FitNesse/SuiteAcceptanceTests/SuiteSlimTests/TestSystemSlimSuite/TimeoutInSlimTestSystem/content.txt

```diff
@@ -1,13 +1,14 @@
-For this test to perform we reduce the timeout to 2 seconds. If the server is not responding in time, we should raise an error.
+For this test to perform we reduce the timeout to 1 seconds. If the server is not responding in time, we should raise an error.
 
-#!define slim.flags {-s 12}
-#|fitnesse.fixtures.Sleep|20000|
 
 !| script                        |
 |given page|TestPage|with content|${SUT_PATH} !-
 !define TEST_SYSTEM {slim}
-!define slim.flags {-s 2}
-|fitnesse.fixtures.Sleep|20000|
+!define slim.flags {-s 1}
+#Will pass
+|fitnesse.fixtures.Sleep|20|
+
+#Will fail
+|fitnesse.fixtures.Sleep|90000|
 -! |
-|test results for page|TestPage|should contain|The instruction timed out after 2 seconds|
-|test results for page in debug mode|TestPage|should contain|The instruction timed out after 2 seconds|
+|test results for page in debug mode|TestPage|should contain|The instruction timed out after 1 seconds|
```

Comment:
Shouldn't we also have an assertion that the first one does pass (like you state in the comment)?